package.json

```diff
@@ -31,9 +31,9 @@
   },
   "homepage": "https://github.com/insulineru/ai-commit#readme",
   "dependencies": {
-    "chatgpt": "^5.0.0",
-    "dotenv": "^16.0.3",
+    "chatgpt": "^5.2.5",
+    "dotenv": "^16.4.5",
     "gpt-3-encoder": "^1.1.4",
-    "inquirer": "^9.1.4"
+    "inquirer": "^11.1.0"
```

Comment:
do we really need to bump 2 major versions there? is it still work?

Reply:
when I try in list mode, everything works fine 👍

Reply:
I believe it's crucial to keep dependencies updated to maintain security and stability in our projects, at least that's the way I do it